### DISCUSSION TITLE
google analytics anonimize IP

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#7405] [Ideas]
Hi all, 

If exist some interesting about the possibility to add an option about the enable and disable of the google analytics anonymize IP functionality I can do a PR to add this option to the `OrchardCore.Google` module.

Nothing complex, a simple checkbox that enable IP anonymize over all site.

I'm waiting about your feedback.
Thank you.

**Comment by jptissot** [comment]
I see no issue with adding a feature like this.

**Reply by PiemP** [reply]
it's an internal poll...I try to understand if exist some interest about this option because I have some interest about it...I don't know the right way to create a proposal: if I have to open an issue or if I have to open a discussion...I have opened a discussion...sorry if I have made the wrong choice 😓
